README.md

```diff
@@ -100,6 +100,11 @@ state change events from that device. You can use the following steps to export
 the MQTT logs from [Stackdriver](https://cloud.google.com/stackdriver/),
 which will report connection state changes.
 
+1. Enable logging for IoT Core Registry.
+   *  Navigate to IoT Core Registries page in the Cloud Console.
```

Comment:
I agree with you on being consistent Dave, but https://console.cloud.google.com/iot/registries is not opening the intended page. The link to go to the Registries page directly has Project ID in it. Example: https://console.cloud.google.com/iot/registries?project=my-gcp-project. Otherwise, it is going to *Google Cloud IoT API* page. This happens even if you have an active session to GCP available. I thought it might confuse. 
Please let me know your thoughts.

Reply:
Sure. Thank you.